blag/markdown.py

```diff
@@ -85,6 +86,7 @@ def convert_markdown(
         tags = meta["tags"].split(",")
         tags = [t.lower() for t in tags]
         tags = [t.strip() for t in tags]
+        tags = [t.replace(" ","-") for t in tags]
```

Comment:
Thise change seems reasonable, can you write a test case that ensures that the replace is working?